bindings/rust/src/lib.rs

```diff
@@ -1629,6 +1724,50 @@ macro_rules! sig_variant_impl {
                 assert_ne!(pk_uncomp2.unwrap(), pk);
                 assert_ne!(pk_deser2.unwrap(), pk);
             }
+
+            #[cfg(feature = "serde")]
```

Comment:
I would suggest to add -features=serde to [ci.yml](https://github.com/supranational/blst/blob/master/.github/workflows/ci.yml#L51).